streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -79,11 +79,14 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
 
 public class KStreamImplTest {
 
     private final Consumed<String, String> stringConsumed = Consumed.with(Serdes.String(), Serdes.String());
+    @SuppressWarnings("deprecation")
+    private final org.apache.kafka.streams.kstream.Serialized<String, String> stringSerialized =
+        org.apache.kafka.streams.kstream.Serialized.with(Serdes.String(), Serdes.String());
```

Comment:
Cannot use import as we cannot suppress deprecation warning on import (similar in the tests).